### PR TITLE
Send prolonged rate limiting messages to logstash

### PR DESCRIPTION
**Pull request by brandon-m-skinner**
## Description

Currently we only send prolonged rate limiting messages to Slack. This should also go to logstash for debugging purposes.

## Checklist

Please make sure the items below have been covered before requesting a review:

- [x] This change works and has been tested locally (or has an appropriate fallback).
- [x] This change works and has been tested on a Go sandbox.
- [x] This change has relevant unit tests (if applicable).
- [x] This change has relevant documentation additions / updates (if applicable).

## Steps to Test

1. Set the rate limiting thresholds for Search queries and indexing low.
2. Trigger ratelimiting. There should be nothing related sent to logstash.
3. Apply PR.
4. Trigger ratelimiting. A logstash entry should be created.

### CODE REVIEW

File: search/includes/classes/class-queue.php [hunk 1]
```diff
@@ -721,13 +721,21 @@ public function maybe_alert_for_prolonged_index_limiting() {
 		}
 
 		$message = sprintf(
-			'Application %d - %s is index limited for %d seconds',
+			'Application %d - %s has had it\'s Elasticsearch indexing rate limited for %d seconds. Large batch indexing operations are being queued for indexing in batches over time.',
```
Comment (nickdaugherty) [inline search/includes/classes/class-queue.php:724]:
```suggestion
			'Application %d - %s has had its Elasticsearch indexing rate limited for %d seconds. Large batch indexing operations are being queued for indexing in batches over time.',
```

File: search/includes/classes/class-search.php [hunk 2]
```diff
@@ -691,13 +691,21 @@ public function maybe_alert_for_prolonged_query_limiting() {
 		}
 
 		$message = sprintf(
-			'Application %d - %s is query limited for %d seconds',
+			'Application %d - %s has had it\'s Elasticsearch queries rate limited for %d seconds. Half of traffic is diverted to the database when queries are rate limited.',
```
Comment (nickdaugherty) [inline search/includes/classes/class-search.php:694]:
```suggestion
			'Application %d - %s has had its Elasticsearch queries rate limited for %d seconds. Half of traffic is diverted to the database when queries are rate limited.',
```

**Review by nickdaugherty — DISMISSED** [review]
Looks good, just two small grammar tweaks.

**Review by netsuso — APPROVED** [review]

**Comment by netsuso** [comment]
stacks-r1279

**Pull request merged**